django/urls/resolvers.py

```diff
@@ -405,7 +405,15 @@ def _check_custom_error_handlers(self):
         # All handlers take (request, exception) arguments except handler500
         # which takes (request).
         for status_code, num_parameters in [(400, 2), (403, 2), (404, 2), (500, 1)]:
-            handler, param_dict = self.resolve_error_handler(status_code)
+            try:
+                handler, param_dict = self.resolve_error_handler(status_code)
+            except (ImportError, ModuleNotFoundError, ViewDoesNotExist) as e:
```

Comment:
`ModuleNotFoundError` is a subclass of `ImportError`, so we could simplify this to `except (ImportError, ViewDoesNotExist)`.

Reply:
Right, Thanks!